src/router/index.js

```diff
@@ -1,48 +0,0 @@
-import { createRouter, createWebHashHistory } from 'vue-router'
-import Home from '../views/Home.vue'
-import About from '../views/About.vue'
-import FromMeToGmail from '../views/graphs/FromMeToGmail.vue'
-import FromMeToNonGmail from '../views/graphs/FromMeToNonGmail.vue'
-import ToMeFromGmail from '../views/graphs/ToMeFromGmail.vue'
-import ToMeFromNonGmail from '../views/graphs/ToMeFromNonGmail.vue'
-
-const routes = [
-  {
-    path: '/',
-    name: 'Home',
-    component: Home
-  },
-  {
-    path: '/docs',
-    name: 'Docs',
-    component: About
-    // () => import('../views/About.vue') for lazy loading
-  },
-  {
-    path: '/graphs/frommetogmail',
```

Comment:
I've heard best practice is to snake case paths for readability

Reply:
Thanks for the tip! I'll have this up in my next push.